recipes/ckanweb-deploy-exts.rb

```diff
@@ -117,305 +115,300 @@
 	apprevision ||= app['app_source']['url'][/@(.*)/].sub '@', ''
 	apprevision ||= "master"
 
-	# Don't install extensions not required by the batch node
+	apprelease = app['app_source']['url'].sub('http', "git+http").sub(/@(.*)/, '')
```

Comment:
I've seen you use include scripts for opsworks/chef, could this be pulled out or not required if it is also being run on the ckan-setup section?

Reply:
This is being run for each extension app. It can't just reuse the value from the CKAN core app.

Reply:
was not meaning just this but the entire extension installation process is on both ckan web and ckan batch

Reply:
Offhand I'm not sure how to define a parameterised recipe. I would use that in several places in our cookbook if I knew an easy way.

Reply:
Okay, https://docs.chef.io/custom_resources/ looks promising. Do you want me to look into applying that before we merge, or I can raise a separate pull request afterward?

Reply:
go seperate